CMakeLists.txt

```diff
@@ -99,6 +99,16 @@ if (USE_TIME_TRACE)
 	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ftime-trace")
 endif()
 
+option(USE_ASAN "Use -fsanitize=asan when compiling (requires Clang)" OFF)
```

Comment:
-fsanitize=address?

Reply:
Yep, I overlooked that - this originally also had `-fsanitize=thread` as an option until I realized our codebase uses too many non-pthread synchronization primitives.